CONTRIBUTING.md

```diff
@@ -64,3 +64,30 @@ Then you just add a line to every git commit message:
 Use your real name (sorry, no pseudonyms or anonymous contributions.)
 
 If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
+
+## Contribution process
+
+### Development
+
+ - Create a personal fork of the project on GitHub.
+ - Use **sycl** branch as baseline for your changes.
+ - Implement/fix your feature (follow [LLVM coding standards](https://llvm.org/docs/CodingStandards.html)).
```

Comment:
Fix your feature seems strange for me :) It look like that we can obviously commit feature which is not working.
How about "Prepare your patch"?